### PR TITLE
User Story #33

### LINKED ISSUE #39: User Story 33, User Profile displays Orders
```
As a registered user
When I visit my Profile Orders page, "/profile/orders"
I see every order I've made, which includes the following information:
- the ID of the order, which is a link to the order show page
- the date the order was made
- the date the order was last updated
- the current status of the order
- the total quantity of items in the order
- the grand total of all items for that order
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #41

child of #79

depends on #57

### PR DESCRIPTION
**Pull request by timnallen**
As a registered user
When I visit my Profile Orders page, "/profile/orders"
I see every order I've made, which includes the following information:
- the ID of the order, which is a link to the order show page
- the date the order was made
- the date the order was last updated
- the current status of the order
- the total quantity of items in the order
- the grand total of all items for that order

closes #39

### CODE REVIEW

**Comment by timnallen** [comment]
Hey guys. Made two order instance methods to get this user order index page up. 

#grand_total = returns the total price of an entire order
#quantity_of_items = returns total quantity

Probably needs a sad path test still, and I just threw the data on the view. Still needs styling and probably to convert the price into currency.

**Review by allieIsOkay — APPROVED** [review]
Looks good, thanks Tim!
I'll open a new issue to track that we get the information on this formatted, but this is more than enough for now.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -15,8 +15,16 @@ def current_user
...
+
   def current_admin?
     current_user && current_user.admin?
   end
+
+  def require_registered
+    render file: '/public/404' unless current_registered?
+  end
+
+  def current_registered?
```
Comment (csvlewis) [inline app/controllers/application_controller.rb:27]: I like your use of helper methods in the authorization!

**Review by csvlewis — APPROVED** [review]
This all looks solid, Tim! Maybe we can work on the formatting of the order total and things like that when we start going over styling and layout of the pages.

**Review by teresa-m-knowles — APPROVED** [review]
Good work! We can work on the styling later
